core/src/test/java/com/graphhopper/reader/osm/OSMReaderTest.java

```diff
@@ -318,26 +318,9 @@ public void testFoot() {
     @Test
     public void testNegativeIds() {
         String fileNegIds = "test-osm-negative-ids.xml";
-        GraphHopper hopper = new GraphHopperFacade(fileNegIds).importOrLoad();
-        Graph graph = hopper.getBaseGraph();
-        assertEquals(4, graph.getNodes());
-        int n20 = AbstractGraphStorageTester.getIdOf(graph, 52);
-        int n10 = AbstractGraphStorageTester.getIdOf(graph, 51.2492152);
-        int n30 = AbstractGraphStorageTester.getIdOf(graph, 51.2);
-        assertEquals(GHUtility.asSet(n20), GHUtility.getNeighbors(carOutExplorer.setBaseNode(n10)));
-        assertEquals(3, GHUtility.count(carOutExplorer.setBaseNode(n20)));
-        assertEquals(GHUtility.asSet(n20), GHUtility.getNeighbors(carOutExplorer.setBaseNode(n30)));
-
-        EdgeIterator iter = carOutExplorer.setBaseNode(n20);
-        assertTrue(iter.next());
-
-        assertTrue(iter.next());
-        assertEquals(n30, iter.getAdjNode());
-        assertEquals(93147, iter.getDistance(), 1);
-
-        assertTrue(iter.next());
-        assertEquals(n10, iter.getAdjNode());
-        assertEquals(88643, iter.getDistance(), 1);
+	    assertThrows(RuntimeException.class, () -> {
+	    	new GraphHopperFacade(fileNegIds).importOrLoad();
+	    });
```

Comment:
Can we assert the error message here so we can be sure the exception was actually thrown because of the negative ID and not for some other reason (like file not found etc.)